### ISSUE TITLE
Unable to compile under Linux

### ISSUE THREAD

**Issue by eduperez** [Beep6581/RawTherapee#3290]
I tried to compile the GTK3 branch under Fedora 23, following the instructions at RawPedia, but there seems to be something wrong with the `cmake` file:

`git clone https://github.com/Beep6581/RawTherapee ~/repo-rt`
`cd ~/repo-rt`
`git checkout remotes/origin/gtk3`
`rm -rf build`
`make clean`
`./clean`
`mkdir build`
`cd build`
```
cmake -DCMAKE_CXX_FLAGS="-std=c++11 -Wno-deprecated-declarations -Wno-unused-result" \
>       -DWITH_LTO="OFF" \
>       -DCMAKE_BUILD_TYPE="release" \
>       -DPROC_TARGET_NUMBER="2" \
>       -DBUILD_BUNDLE="ON" \
>       -DBINDIR="." \
>       -DDATADIR="." \
>       -DCACHE_NAME_SUFFIX="4" ..
```
`make -j6 install`
```
CMake Error at /home/eperez/repo-rt/AboutThisBuild.cmake:27 (string):
  string sub-command REGEX, mode REPLACE needs at least 6 arguments total to
  command.


CMakeFiles/AboutFile.dir/build.make:57: recipe for target 'CMakeFiles/AboutFile' failed
make[2]: *** [CMakeFiles/AboutFile] Error 1
CMakeFiles/Makefile2:67: recipe for target 'CMakeFiles/AboutFile.dir/all' failed
make[1]: *** [CMakeFiles/AboutFile.dir/all] Error 2
Makefile:127: recipe for target 'all' failed
make: *** [all] Error 2

```

I changed all instances of `${GIT_VERSION_WHOLE}` to `"${GIT_VERSION_WHOLE}"` in the `string` calls, and this seems to have fixed the issue.

**Comment by ClaesGitHub** [comment]
Hi!
I use another checkout terminology. Try

> git checkout gtk3

It works for me.

**Comment by eduperez** [comment]
@ClaesGitHub : Tried that, but got the same result; thanks anyway.

**Comment by aserebryakov** [comment]
@eduperez, what CMake version do you use?

**Comment by ClaesGitHub** [comment]
@eduperez I had to build a new Virtualbox today for some experiments, and I fed it with a very minimal Fedora 23. Then I compiled the gtk3 branch of RT - and **it worked just fine**. I followed the standard instructions for Fedora on Rawpedia.  (My cmake version is 3.4.1.)

**Comment by eduperez** [comment]
I'm using cmake 3.4.1, too; this is weird, I do not remember doing anything special about cmake in this machine, everything should be stock around here.

**Comment by eduperez** [comment]
More info: this are the contents of the `AboutThisBuild.txt` file:
```
Branch: 
Version: 
Changeset: 
Compiler: cc 5.3.1
Processor: x86_64
System: Linux
Bit depth: 64 bits
Gtkmm: V3.18.1
Build type: release
Build flags: -std=c++11 -Wno-deprecated-declarations -Wno-unused-result -std=gnu++11 -march=native -Werror=unused-label -fopenmp -Werror=unknown-pragmas -O3 -DNDEBUG
Link flags:  -march=native
OpenMP support: ON
MMAP support: ON
```

Notice how all git-related info is missing.

**Comment by eduperez** [comment]
I found it! Launching with the original `AboutThisBuild.cmake` file, the `make` command produces the message "`-- git command found: /home/eperez/git`"  (sorry, I did not notice that before).

There is effectively a `git` _folder_ in my home, but it is not the _executable_ that `cmake` is looking for. Culprit is at "`find_file(GIT_CMD git)`" inside `AboutThisBuild.cmake`, because hat is not a proper method to find an executable in a Linux machine; changing that line to "`find_program(GIT_CMD git)`" solved the issue.
